modules/serverless-resolving-operator-upgrade-failure.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * /serverless/admin_guide/serverless-upgrades.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-resolving-operator-upgrade-failure_{context}"]
+= Resolving {ServerlessOperatorName} upgrade failure
+
+In some cases, such as when you do manual uninstalls and reinstalls, you might encounter an error when upgrading {ServerlessOperatorName}. In this situation, you need to manually reinstall {ServerlessOperatorName}.
+
+.Procedure
+
+. Identify the version of {ServerlessOperatorName} that was installed originally using the `project.yaml` files for the different releases.
++
+For example, the error message during attempted upgrade contains this string:
++
+[source,yaml]
+----
+The installed KnativeServing version is v0.25.1.
```

Comment:
Let's update the versions to something more recent, for example `v1.5.0`.  (which corresponds to Serverless 1.26.0). I will provide examples below for this version.